pipeline_reviews/transform.py

```diff
@@ -0,0 +1,92 @@
+"""Validates received review inputs"""
+
+from datetime import datetime
+from os import environ
+
+import pandas as pd
+from pandas import DataFrame
+from dotenv import load_dotenv
+from psycopg2.extras import RealDictCursor
+from psycopg2 import connect
+from psycopg2.extensions import connection
+
+
+def get_db_connection() -> connection:
```

Comment:
could we have a try and except just in case it doesn't connect to the database here